packages/nodes-base/nodes/Okta/UserDescription.ts

```diff
@@ -0,0 +1,524 @@
+import type { INodeProperties } from 'n8n-workflow';
+const BASE_API_URL = '/api/v1/users/';
+export const userOperations: INodeProperties[] = [
+	{
+		displayName: 'Operation',
+		name: 'operation',
+		type: 'options',
+		noDataExpression: true,
+		displayOptions: {
+			show: {
+				resource: ['user'],
+			},
+		},
+		options: [
+			// Create Operation
+			{
+				name: 'Create',
+				value: 'create',
+				description: 'Create a new user',
+				routing: {
+					request: {
+						method: 'POST',
+						url: BASE_API_URL,
+						qs: { activate: '={{$parameter["activate"]}}' },
+						returnFullResponse: true,
+					},
+				},
+				action: 'Create a new User',
+			},
+			// Delete Operation
+			{
+				name: 'Delete',
+				value: 'delete',
+				description: 'Delete an empty Bucket',
+				routing: {
+					request: {
+						method: 'DELETE',
+						url: '={{"/api/v1/users/" + $parameter["userId"]}}',
+						returnFullResponse: true,
```

Comment:
This returns an empty item, We should maybe return something like `success: true` like we do in other nodes, Should we also add the `sendEmail` property as it is the only other API option for this operation

Reply:
this fixed now and added sendEmail field too